newshour-chyron/readme.md

```diff
@@ -1 +1,107 @@
-***Needs to be updated***
+# Chyrons
+
+## Project Overview
+This project provides a dataset used for the detection of chyrons/banner-text that appears onscreen in video media.  
+[Chyron](https://www.merriam-webster.com/dictionary/chyron) - a caption superimposed over *usually* the lower part of a video image (as during a news broadcast). Note, not neccesarily lower third. Sometimes they are temporary and will fade/blink in and out. 
+Named after the company that made this technology. aka: the text that appears under people during broadcasts.  
+### Specs
+* Annotation project name - newshour-chyron
+* Annotator Demographics
+    * Number of annotators - 1 
+    * Occupational description - College Student
+    * Age range - 20s
+    * Education - College
+* Annotation Environment information
+    * Name - VIA
+    * Version - Unknown
+    * Link/Tool Used/User Manual - (See below Tool Installation)
+* Project changes
+    * Number of Batches - 1, (The batch is named batch2 likely as a name - there seems to be only one batch used.)
+    * Other Version Control Information - Unknown/1.0
+
+## Tool Installation: VIA tool
+_The following section is recalled from memory and may not be fully accurate._  
+[VIA3](https://www.robots.ox.ac.uk/~vgg/software/via/). This project used [this version](https://github.com/kelleyl/clams-via3/blob/master/app.py#L16) and the Video Annotator tool.  
+A video must be imported. Docker is likely used.  
+
+## Annotation Guidelines
+> This `readme.md` is the guideline for this project. 
+
+### Preparation
+Videos are loaded into the VIA tool for annotation. 
+The annotator watches a video using the annotation tool, and labels chyrons in time.  
+
+### What to Annotate
+Usually, chyrons will fade in and out.  
+**START TIME** - When a chyron has fully appeared/solid and is no longer see through, pause the video and highlight the chyron via a bounding box. It's not too dissimilar from taking a screenshot.  
+**TEXT** - When the chyron is highlighted, double-click it and in the DESCRIPTION box, copy/write out what was said in the chyron exactly as it was displayed. When a chyron consists of more than one line, use the enter/return key (becomes \n in the gold output) to make a new line in the description.  
+**END TIME** - After that, submit how long the chyron was onscreen: annotating the end time as when it begins to become transparent.  
+_Time format: (sec.ten-thousandths of sec)_   
+
+### How to Annotate It
+Please see VIA link for tool usage on step by step how to use the tool. 
+
+### Decisions, Differentiation, and Precision Level during Annotation
+Some decisions within the annotation process are described above.  
+
+> Precision Details
```

Comment:
any particular reason to use quoting instead of sub-level heading...?

Reply:
In my opinion, these "edge-case" handling should be a part of the guidelines (not a sibling).

Reply:
Just to check, you mean basically that it should be a ### subcategory within ## Annotation Guidelines. 

Yes, will use ####.